### PR TITLE
Shopware

### PR DESCRIPTION
**Pull request by Crell**
This seems to work.  Some bits are borrowed from a customer project so I'm not 100% certain they're all needed, but I'm inclined to trust them.

cf: https://github.com/platformsh-templates/shopware/pull/1

### CODE REVIEW

File: templates/shopware/.platform.template.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+# Templates are a small amount of information supporting a template URL. Each template is selectable at the project-creation step.
+info:
+  # Unique machine name, prefaced by a vendor or organization identifier
+  id: platformsh/shopware
+  # The human-readable name of the template.
+  name: Shopware
+  # Human-readable descriptive text for the template. Supports limited HTML.
+  description: |
+      <p>This template provides a basic Symfony 5 skeleton.  It is configured for Production mode by default so the usual Symfony "welcome" page will not appear.  That can be adjusted in <code>.platform.app.yaml</code>.</p>
```
Comment (chadwcarlson) [inline templates/shopware/.platform.template.yaml]: Update description

File: templates/shopware/.platform.template.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      content: "PHP 7.3<br/>MariaDB 10.4"
+
+# This key describes the initialization call made to the master environment at
+# project creation time. This is part of the full v2 UI operation mode, which
+# places project schema/options selection early in the creation process, rather
+# than later as it exitss now. To allow this schema to be backwards-compatible,
+# this key also gets mapped to the appropriate location in project.settings so
+# that the current UI can have its own workflow overridden as well.
+initialize:
+  repository: git://github.com/platformsh-templates/symfony5.git@master
```
Comment (chadwcarlson) [inline templates/shopware/.platform.template.yaml]:
```suggestion
  repository: git://github.com/platformsh-templates/shopware.git@master
```

File: templates/shopware/.platform.template.yaml (hunk 1 above)
Comment (chadwcarlson) [reply templates/shopware/.platform.template.yaml]:
```suggestion
      <p>This template builds Shopware on Platform.sh using Composer.</p>
```

File: templates/shopware/.platform.template.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+# Templates are a small amount of information supporting a template URL. Each template is selectable at the project-creation step.
+info:
+  # Unique machine name, prefaced by a vendor or organization identifier
+  id: platformsh/shopware
+  # The human-readable name of the template.
+  name: Shopware
+  # Human-readable descriptive text for the template. Supports limited HTML.
+  description: |
+      <p>This template provides a basic Symfony 5 skeleton.  It is configured for Production mode by default so the usual Symfony "welcome" page will not appear.  That can be adjusted in <code>.platform.app.yaml</code>.</p>
+      <p>Symfony is a high-performance loosely-coupled PHP web development framework.</p>
```
Comment (chadwcarlson) [inline templates/shopware/.platform.template.yaml]:
```suggestion
      <p>Shopware is a PHP-based ecommerce system.</p>
```

**Comment by Crell** [comment]
Jonas is handling this elsewhere, this PR is no longer needed.

**Pull request closed without merging**